app/javascript/react/components/presentational/ScarcCoverflow/MediaGrid.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react"
+import PropTypes from "prop-types"
+
+function GridList({collections, selected_class, slide_clicked, on_load_all_images}) {
+  return(
+    <ul>
+      {collections.map((slide, i) => {
+        return (
+          <li key={`collection.${i}`} className={ `${selected_class(i)}` } data-index={i}>
+            <div className="selected-collection-header">
+              <h1 className="collection-name">{slide.collection.name}</h1>
+              <p className="collection-description">{slide.collection.detail}</p>
+            </div>
+            <GridItem
+              slides={slide.collection.slides}
+              slide_clicked={slide_clicked}
+              on_load_all_images={on_load_all_images}
+              index={i}
+            />
+          </li>
+        )
+      })}
+    </ul>
+  );
+}
+
+function GridItem({slides, slide_clicked, on_load_all_images, index}) {
+  return(
+    <div className={`grid-content grid-${index}`}>
+      {slides.map((s, j) => {
+        return (
+          <div
+            className={"grid-item"}
+            key={`slide.${index}.${j}`}
+            onClick={_e => slide_clicked(j)}
+            style={{
+              width: "300px",
+              marginBottom: "30px"
+            }}
+          >
+            <img
+              src={s.large}
+              style={{
+                width: "100%"
+              }}
+              onLoad={() => { on_load_all_images(index) }}
+            />
+          </div>
+        )
+      })}
+    </div>
+  );
+}
+
+class MediaGrid extends Component {
```

Comment:
Could we use the `const MediaGrid = props => {}` syntax?
Just to be consistent with the rest of the components in this PR/Kiosk

Reply:
That's a good idea. It looks cleaner, I updated it to use the const syntax.